src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,19 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int ARRAY_SIZE = 6;
 
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[ARRAY_SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < ARRAY_SIZE; i++) {
+            if (i < 3) {
```

Comment:
rely on your constant. If you decide to change it, you may forget to change it here